services/nublado2/values-bleed.yaml

```diff
@@ -0,0 +1,49 @@
+nublado2:
+  jupyterhub:
+    debug:
+      enabled: true
+
+    hub:
+      image:
+        name: lsstsqre/nublado2
+        tag: "1.0.0"
+
+    singleuser:
+      storage:
+        type: none
+
+    auth:
+      state:
+        cryptoKey: "080c6d0eb8d9f84ef5f0860bea71dba6ab2e1be712c7bc8c2eaf92321c9fd968"
+
+    proxy:
+      secretToken: "3854245f5b48412ee9b7a2c985da6acdc146f80cc4f10eb175f0efa24530399b"
+      service:
+        type: ClusterIP
+
+    ingress:
+      hosts: ["bleed.lsst.codes"]
+      annotations:
+        nginx.ingress.kubernetes.io/auth-signin: "https://bleed.lsst.codes:31337/login"
+        nginx.ingress.kubernetes.io/auth-url: "http://gafaelfawr-service.gafaelfawr.svc.cluster.local:8080/auth?scope=exec:notebook"
+
+  config:
+    options_form:
+      images_url: "http://cachemachine.cachemachine.svc.cluster.local/cachemachine/jupyter/available"
```

Comment:
This should be in the chart rather than lsp-deploy, I think, because it's not going to vary by environment.

As I learned the hard way with Gafaelfawr, the Chilean clusters don't use `.svc.cluster.local` as their domain name. Omitting that seems to work.

Reply:
UGH!  That's really annoying.  Thanks for mentioning this.  You'll probably mention it a few more times before I forget.  Will do on adding it to the chart.